pipelines/common/src/test/java/com/google/fhir/analytics/ParquetUtilTest.java

```diff
@@ -69,14 +68,14 @@ public class ParquetUtilTest {
 
   private FhirContext fhirContext;
 
-  private FileSystem fileSystem;
-
   private Path rootPath;
 
+  @Rule public TemporaryFolder testFolder = new TemporaryFolder();
+
   @Before
   public void setup() throws IOException {
-    fileSystem = Jimfs.newFileSystem(Configuration.unix());
```

Comment:
IIRC the main benefit of using `Jimfs` is to avoid accessing disk in unit-tests (and instead use an in-memory filesystem).  Is there a particular reason that we cannot use this anymore?  If this change is for Windows cases, according to [here](https://github.com/google/jimfs/blob/a62ae2c2bdc7b28e1e3bc857ae47e01150445aba/jimfs/src/main/java/com/google/common/jimfs/Jimfs.java#L52) it seems that `Jimfs` should support Windows too, doesn't it? In fact, if that is the case, this might be a way to address my other concern about not testing Windows in our CI, WDYT?

Reply:
I faced some complications while using the Jimfs filesystems as I could not pass the mocked FileSystem to `ParquetUtil`. Also, since the current code is refactored to not use `java.nio.file.*`, it was not helpful to use Jimfs . Hence the test cases has been refactored to use the underlying platform file system and leave it to beam to infer correctly. Though the files are created on disk, they should be auto deleted after the tests complete.